src/core/lib/slice/slice_utils.h

```diff
@@ -27,7 +27,9 @@
 
 #include "src/core/lib/gpr/murmur_hash.h"
 
+namespace grpc_core {
 extern uint32_t g_hash_seed;
```

Comment:
Does this need to be declared in both slice_internal.h and slice_utils.h?  Can we eliminate one of them?

Reply:
Eliminated the one in slice_internal.h.